docs/src/modules/components/MarkdownElement.js

```diff
@@ -120,14 +120,26 @@ const Root = styled('div')(({ theme }) => ({
     },
     '&:hover .anchor-link-style': {
       display: 'inline-block',
-      padding: '0 8px',
+      textAlign: 'center',
+      lineHeight: '23px',
+      marginLeft: 10,
+      height: '26px',
+      width: '26px',
+      background:
+        theme.palette.mode === 'dark'
+          ? alpha(theme.palette.primary[800], 0.3)
+          : theme.palette.primary[50],
+      border: '1px solid',
+      borderColor:
+        theme.palette.mode === 'dark' ? theme.palette.primaryDark[500] : theme.palette.grey[200],
+      borderRadius: 8,
       color: theme.palette.text.secondary,
       '&:hover': {
         color: theme.palette.text.primary,
       },
       '& svg': {
-        width: '0.7em',
-        height: '0.7em',
+        width: '0.875rem',
+        height: '0.875rem',
         fill: 'currentColor',
       },
```

Comment:
I'm not sure that it feels aligned:

<img width="400" alt="Screenshot 2022-02-18 at 00 13 23" src="https://user-images.githubusercontent.com/3165635/154587871-3ef18b79-e84e-494c-a2de-622dbf00d894.png">


Maybe we could handle this in a different PR? As a side note, I think that we could get rid of the "anchor-link" class and use `scroll-margin-top` instead, simpler, fewer DOM nodes.

Reply:
We can refine it for sure in a follow-up PR!